internal/impl/devbox.go

```diff
@@ -714,7 +714,7 @@ func (d *Devbox) computeNixEnv(ctx context.Context, usePrintDevEnvCache bool) (m
 		}
 		env[key] = val
 	}
-	currentEnvPath := env["PATH"]
+	currentEnvPath := envir.GetPath(env)
```

Comment:
I don't know if `GetPath` is adding much value here, over `env[envir.Path]`. 

`env[envir.Path]` feels easier to read since one can readily understand what it is doing.

What's the motivation?

Reply:
Yeah, it seems a little over-wrapping.

I want to limit the adding/getting/removing/updating one specific environment variable operations in functions so that callers use functions instead of envs.